src/Cms/Page.php

```diff
@@ -830,9 +829,8 @@ public function modified(
 		string|null $handler = null,
 		string|null $languageCode = null
 	): int|string|false|null {
-		$modified = $this->storage()->modified(
-			VersionId::default($this),
-			$languageCode
+		$modified = $this->version()->modified(
+			$languageCode ?? 'current'
```

Comment:
`Thought`: v4 seems to be using the `default` language, so this would technically be a breaking change. However I think it makes sense to go with `current` here. But then we need to document it in the release notes and also think about consistent behavior in `$file->modifiedContent()`.

Reply:
I think we are really cleaning things up here by following the scheme to always use `current` for all model methods. I agree that we need to follow through here and do the same in modifiedContent then. I will change it and add the release notes and breaking changes.